wiki/Announcements/osu!academy/en.md

```diff
@@ -1,61 +1,56 @@
-osu!academy
-==============
+# osu!academy
 
 ![osu!academy's logo](/wiki/Announcements/osu!academy/img/Osu!academy_base.png "osu!academy's logo")
 
-Introduction
----------------
-
-Source: http://news.ppy.sh/post/70504430082/introducing-the-osu-academy
-Official Youtube channel: http://www.youtube.com/user/osuacademy/videos
-Official thread: http://osu.ppy.sh/forum/t/169707
-
 **osu! academy** aims to give a comprehensive look at all aspects of osu!, from playing to mapping and everything in-between.
 
 We hope that new players will find these videos informative, and help ease the arguably steep learning curve associated with the game. If you have any questions or comments, please forward them to [ztrot](//osu.ppy.sh/u/6347); he is the primary and almost sole owner of the osu! academy and its channel.
```

Comment:
hmm, it seems like this was missed 68 times in 43 files of the whole repo (well, at least in this branch)